src/components/Modal/ProposalPlugins.vue

```diff
@@ -79,8 +85,13 @@ export default {
       return `https://raw.githubusercontent.com/snapshot-labs/snapshot.js/master/src/plugins/${plugin}/logo.png`;
     }
 
-    function showButton(plugin) {
-      return plugin.name !== 'SafeSnap';
+    function showButton(pluginObj) {
```

Comment:
You can hide the full block, not just the button

Reply:
Hm, are you sure we don't want some way to see the plugins that will be used on proposal creation?

Reply:
If so I would need to redesign a lot of this component